### PR TITLE
readme: clarify goals a bit more

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,22 +11,24 @@ Fizzy aims to be a fast integer-only WebAssembly interpreter written in C++.
...
-- Support an efficient big integer API (256-bit and perhaps 384-bit)
-- Support runtime metering in the interpreter
-- Support enforcing a call depth bound
-- Further restrictions of complexity (e.g. number of locals, number of function parameters, number of labels, etc.)
+- [ ] Support an efficient big integer API (256-bit and perhaps 384-bit)
+- [ ] Support optional runtime metering in the interpreter
+- [ ] Support enforcing a call depth bound
+- [ ] Further restrictions of complexity (e.g. number of locals, number of function parameters, number of labels, etc.)
+
+The goals ticked are already met by Fizzy.
```
Comment (chfast) [inline README.md:31]: Obvious?

File: README.md [hunk 2]
```diff
@@ -11,22 +11,24 @@ Fizzy aims to be a fast integer-only WebAssembly interpreter written in C++.
 ## Goals
 
 I) Code quality
-- Clean and modern C++17 codebase
-- Easily embeddable (*and take part of the standardisation of the "C embedding API"*)
+- [x] Clean and modern C++17 codebase
```
Comment (chfast) [inline README.md:14]:
```suggestion
- [x] Clean and modern C++17 codebase without external dependencies
```

File: README.md (hunk 1 above)
Comment (axic) [reply README.md:31]: Is it? I can remove it.

Comment (axic) [reply README.md:31]: Also wasn't sure what to do with spectests, because we partially meet that goal.

**Pull request merged**